### DISCUSSION TITLE
Reddit ban

### DISCUSSION THREAD

**Discussion by valinet** [valinet/ExplorerPatcher#1741] [Announcements]
So, I’ve been permanently banned from r/Windows, r/Windows10, and r/Windows11 with no warning because some wise “moderators” there can’t take a discussion about the existence of Windows LTSC, it’s forbidden territory, apparently. That’s the level, apparently, tbh, fits well with Windows 11 in general. Anyway, I won’t be able to engage there.

Discussion that led to this: https://www.reddit.com/r/Windows11/comments/15dq3vi/why_file_transfer_screen_is_white_while/

Apparently, they went out of their way to dig for some comment I made in r/Windows10 a while ago so they could ban me there as well: https://www.reddit.com/r/Windows10/comments/13n8pj8/went_back_to_windows_10_feels_great/

The ban on r/Windows is totally random, whatever…

Don’t worry, I live in a formerly communist country, the attitude is familiar, no resentments. That’s the level, as I said.

:/

**Comment by Avnsx** [comment]
Welcome to Reddit degeneracy and its narcissistic moderators 🤣

Most people on there are weird, don't invest too many thoughts into it. Probably one of the mods of those subreddits, moderates all of the others in addition, so he got upset and cross-banned you on all of them.

**Reply by EPFAN** [reply]
I left reddit as a member a long time ago due to moderators' sense of being god-like. That fact and also the crazies who hang out at reddit with nothing better to do than posts comments unrelated to the topics.

**Comment by DealsBeam** [comment]
Welcome to Reddit

**Comment by ryderbelserion** [comment]
That's the Reddit experience!

**Comment by 5T33Z0** [comment]
I've been there as well,… jiust on the Hackintosh side of things. For _supporting_ Clover users. :D

**Comment by Naamloos** [comment]
thanks for the ban kind stranger!

**Comment by Faks** [comment]
Reddit is trash same as Stack Overflow, that is harsh reality they also dislike hard truth were silenced ban there, in the end just deleted profiles from there and moved on...

**Reply by darrylnoakes** [reply]
I resent the insult to SO. Yes, those that are active (mods, committed users, etc.) can seem harsh, but it is largely because there are simply so many less-dedicated users that there is no time for niceties. Many people simply aren't even reading the most basic guidelines; they don't give an MRE in their questions, or explanations in their answers, etc. The active folks aren't being "toxic" to anyone in specific; they just need to be ridiculously efficient and are spending the minimum time to get questions answerable and answers useful.

**Reply by Faks** [reply]
problem is not about toxicity but lack of common sense in many cases instead normally answering or trying at-least to help on some level instead they just lock tread and do down rating and in the end you just can't do anything, my advice just use chat gpt and it is far more useful in many cases, and in rare cases use (DuckDuckGo or google ) and search what you need and combine the information with chat gpt and life can be a bliss.

**Comment by aubymori** [comment]
kkkkkkkk average redditards

**Reply by TheGreenSquare** [reply]
> Original comment in Portuguese

Yeah, sure.

**Comment by craftogrammer** [comment]
Yes, that's real Reddit...

**Comment by Purfview** [comment]
Who cares about that cesspool, just create a new acc using https://temp-mail.org
